src/main/java/persistence/DefaultCatalog.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Everything in the Java directory is generated by Jooq, we could change the directory for more clarity if needed